core/src/main/java/org/apache/calcite/plan/RelOptNode.java

```diff
@@ -45,9 +47,20 @@
    * <p>If you want a descriptive string which contains the identity, call
    * {@link Object#toString()}, which always returns "rel#{id}:{digest}".
    *
+   * @return Digest string of this {@code RelNode}
+   */
```

Comment:
Thanks a lot to keep this `String getDigest`, which is heavily relied on in our internal code base.